great_expectations/experimental/datasources/sql_datasource.py

```diff
@@ -206,7 +355,37 @@ def add_splitter_year_and_month_and_day(
             This sql asset so we can use this method fluently.
         """
         return self._add_splitter(
-            method_name="split_on_year_and_month_and_day", column_name=column_name
+            ColumnSplitterYearAndMonthAndDay(
+                method_name="split_on_year_and_month_and_day", column_name=column_name
+            )
+        )
+
+    def add_splitter_datetime_part(
+        self, column_name: str, datetime_parts: List[str]
+    ) -> _SQLAsset:
+        return self._add_splitter(
+            ColumnSplitterDatetimePart(
+                method_name="split_on_date_parts",
+                column_name=column_name,
+                datetime_parts=datetime_parts,
+            )
+        )
+
+    def add_splitter_column_value(self, column_name: str) -> _SQLAsset:
+        return self._add_splitter(
+            ColumnSplitterColumnValue(
+                method_name="split_on_column_value",
+                column_name=column_name,
+            )
+        )
+
+    def add_splitter_divided_integer(self, column_name: str, divisor: int) -> _SQLAsset:
```

Comment:
@billdirks There is also a "mod integer" splitter in V3 (we would need to check the code and documentation for any others that could be useful candidates for adding here as well).  Thanks.

Reply:
I plan to add the remaining splitters in the next PR. There are a few more.